src/zip.js

```diff
@@ -10,7 +10,7 @@ module.exports = function(gj, options) {
 
     [geojson.point(gj), geojson.line(gj), geojson.polygon(gj)]
         .forEach(function(l) {
-        if (l.geometries.length) {
+        if (l.geometries[0].length) {
```

Comment:
Ah, it is guaranteed in the case of a `POLYGON` type and a `POLYLINE` type, but not for a `POINT`. I'll update the check per your suggestion.